src/prefect/tasks/neo4j/neo4j_tasks.py

```diff
@@ -0,0 +1,243 @@
+            - A `list` of `dict` if `return_result_as` is `json`.
+            - A `pandas.DataFrame` if `return_result_as` is `dataframe`.
+
+        Raises:
+            - `ValueError` if both `server_uri` and `server_uri_env_var`
+                are `None`.
+            - `ValueError` if `server_uri` is `None` and `server_uri_env_var`
+                is not found.
+            - `ValueError` if both `user` and `user_env_var`
+                are `None`.
+            - `ValueError` if `user` is `None` and `user_env_var` is
+                not found.
+            - `ValueError` if both `password` and `password_env_var`
+                are not found.
+            - `ValueError` if `password` is `None` and `password_env_var`
+                is not found.
+            - `ValueError` if `db_name` is `None` and `db_name_env_var`
+                is not found.
+            - `ValueError` if `cypher_query` is `None`.
+            - `ValueError` if `return_result_as` is not one of
+                `raw`, `json`, `dataframe`.
+            - `prefect.engine.signals.FAIL` if any error occurs while establishing
+                the connection with Neo4j.
+            - `prefect.engine.signals.FAIL` if any error occurs while running
+                the Cypher query.
+        """
+        if not server_uri and not server_uri_env_var:
+            msg = "Please provide either the `server_uri` or the `server_uri_env_var`."
+            raise ValueError(msg)
+
+        if not server_uri and server_uri_env_var not in os.environ.keys():
+            msg = f"`{server_uri_env_var}` not found in environment variables."
+            raise ValueError(msg)
+
+        neo4j_uri = server_uri or os.environ[server_uri_env_var]
+
+        if not user and not user_env_var:
+            msg = "Please provide either the `user` or the `user_env_var`."
+            raise ValueError(msg)
+
+        if not user and user_env_var not in os.environ.keys():
+            msg = f"`{user_env_var}` not found in environment variables."
+            raise ValueError(msg)
+
+        neo4j_user = user or os.environ[user_env_var]
+
+        if not password and not password_env_var:
+            msg = "Please provide either the `password` or the `password_env_var`."
+            raise ValueError(msg)
+
+        if not password and password_env_var not in os.environ.keys():
+            msg = f"`{password_env_var}` not found in environment variables."
+            raise ValueError(msg)
+
+        neo4j_password = password or os.environ[password]
+
+        neo4j_db_name = None
+        if db_name:
+            neo4j_db_name = db_name
+        elif db_name_env_var and db_name_env_var not in os.environ.keys():
+            msg = f"`{db_name_env_var}` not found in environment variables."
+            raise ValueError(msg)
+        elif db_name_env_var and db_name_env_var in os.environ.keys():
+            neo4j_db_name = os.environ[db_name_env_var]
```

Comment:
Might as well follow the same pattern as above.
```suggestion
       if not db_name and not db_name_env_var:
            msg = "Please provide either the `db_name` or the `db_name_env_var`."
            raise ValueError(msg)

        if not db_name and db_name_env_var not in os.environ.keys():
            msg = f"`{db_name_env_var}` not found in environment variables."
            raise ValueError(msg)
            
        neo4j_db_name = db_name or os.environ[db_name_env_var]
```

Reply:
Gotcha, I'll resolve this thread then!